interpreter/.gitignore

```diff
@@ -1,13 +1,3 @@
-*.cmo
-*.cmx
-*.native
-*.byte
-*.opt
-*.unopt
-*.js
-*.zip
```

Comment:
You probably want to keep *.js and *.zip

Reply:
Fixed.